docs/sources/features/datasources/x-ray.md

```diff
@@ -0,0 +1,139 @@
+
+| Name                       | Description                                                                                                              |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| Name                     | The data source name. This is how you refer to the data source in panels and queries.                                    |
+| Default                  | Default data source means that it will be pre-selected for new panels.                                                   |
+| Default Region           | Used in query editor to set region. (can be changed on per query basis)                                                   |                                                                    
+| Auth Provider            | Specify the provider to get credentials.                                                                                 |
+| Credentials profile name | Specify the name of the profile to use (if you use `~/.aws/credentials` file), leave blank for default.                  |
+| Assume Role Arn          | Specify the ARN of the role to assume.                                                                                    |
+| External ID              | If you are assuming a role in another account, that has been created with an external ID, specify the external ID here. |
+
+## Authentication
+
+In this section we will go through the different type of authentication you can use for X-Ray data source.
+
+### AWS credentials
+
+If the Auth Provider is `Credentials file`, then Grafana tries to get credentials in the following order:
+
+- Environment variables (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`)
+- Hard-code credentials
+- Shared credentials file
+- IAM role for Amazon EC2
+
+Refer to [Configuring the AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) in the AWS documentation for more information.
+
+### AWS credentials file
+
+Create a file at `~/.aws/credentials`. That is the `HOME` path for user running grafana-server.
+
+> **Note:** If the credentials file in the right place, but it is not working, then try moving your .aws file to '/usr/share/grafana/'. Make sure your credentials file has at most 0644 permissions.
+
+Example credential file:
+
+```bash
+[default]
+aws_access_key_id = <your access key>
+aws_secret_access_key = <your access key>
+region = us-west-2
+```
+
+## Query editor 
+
+The most important field in the editor is the query type. There are four query types:
+- Trace List (Traces in AWS)
+- Trace Statistics
+- Trace Analytics (Analytics in AWS)
+- Insights
+
+{{< docs-imagebox img="/img/docs/v72/x-ray-query-editor.png" max-width="800px" class="docs-image--left" caption="X-Ray query editor" >}}
+
+### Trace List
```

Comment:
Is Trace List capitalized like this in the UI that you are referring to?

Same question for all the other items in this vicinity.